pennylane/templates/transforms.py

```diff
@@ -0,0 +1,20 @@
+from functools import wraps
+from pennylane.tape.tapes import QuantumTape
+
+
+def adjoint(fn):
+    """ "Create a new method that applies the adjoint tape of `fn`.
+
+    Example:
+            `adjoint(fn)(args)` will apply all of the operations executed during fn(args),
+            but in reverse and with each operation adjointed.

```

Comment:
```suggestion
    **Example**
     
     ``adjoint(fn)(args)`` will apply all of the operations executed during ``fn(args)``,
     but in reverse and with each operation adjointed.
```

Reply:
Sphinx doesn't seem to recognize `Example:` for some reason, so we fake it by just bolding it and removing the colon.

Recommend also adding a fully working code example here, with output! https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html#code-examples